doc/source/serve/http-guide.md

```diff
@@ -268,3 +268,24 @@ Here is a list of adapters; please feel free to [contribute more](https://github
 
 ```
 
+## Best practices for HTTP requests
```

Comment:
Should the title indicate that these are best practices for production?
```suggestion
## Best practices for HTTP requests in Production
```

Also, not sure - should this go under a best practices section under Production Guide instead?

Reply:
I would suggest production guide as well!

Reply:
Sounds good, I moved it to the Best Practices page in the Production Guide.